open-build-service-api/src/lib.rs

```diff
@@ -458,6 +458,74 @@ impl RebuildFilters {
     }
 }
 
+#[derive(Clone, Debug, Default)]
+pub struct JobHistoryFilters {
+    packages: Vec<String>,
+    codes: Vec<PackageCode>,
+    limit: Option<usize>,
+}
+
+impl JobHistoryFilters {
+    pub fn all() -> Self {
+        Self::default()
+    }
+
+    pub fn only_package(package: String) -> Self {
+        JobHistoryFilters::all().package(package)
+    }
+
+    pub fn add_package(&mut self, package: String) {
```

Comment:
I copied the awful naming I chose `RebuildFilters::all()` for consistency, since at the time I couldn't think of a better one, but then all of a sudden I did :sweat_smile: so now both got renamed to `::empty()` instead